cdc/model/owner.go

```diff
@@ -238,6 +241,7 @@ func (ts *TaskStatus) RemoveTable(id TableID, boundaryTs Ts, isMoveTable bool) (
 		return nil, false
 	}
 	delete(ts.Tables, id)
+	log.Info("remove a table", zap.Int64("table-id", id))
```

Comment:
can also log `boundaryTs` and `isMoveTable`

Reply:
done